### PR TITLE
Add update to Changelog

### PR DESCRIPTION
**Pull request by ry**

### CODE REVIEW

File: Docs.md [hunk 1]
```diff
@@ -337,7 +337,22 @@ https://github.com/denoland/deno/blob/master/src/msg.fbs
...
+## Changelog
+
+### 2018.11.27 / v0.2.0 / Mildly usable
+
+The resources table was added to abstract various types of I/O streams and other
+allocated state. A resource is an integer identifier which maps to some Rust
+object. It can be used with various ops. Particularly read and write.
+
+Stability and usability improvements. `fetch()` is 90% complete now. Basic REPL
+support was added. Shebang support was added. Command-line argument parsing was
```
Comment (piscisaureus) [inline Docs.md]: set up

File: Docs.md [hunk 2]
```diff
@@ -337,7 +337,22 @@ https://github.com/denoland/deno/blob/master/src/msg.fbs
...
 [CONTRIBUTING.md](https://github.com/denoland/deno/blob/master/.github/CONTRIBUTING.md).
 
-## Change Log
+## Changelog
+
+### 2018.11.27 / v0.2.0 / Mildly usable
+
+The resources table was added to abstract various types of I/O streams and other
+allocated state. A resource is an integer identifier which maps to some Rust
+object. It can be used with various ops. Particularly read and write.
```
Comment (piscisaureus) [inline Docs.md]: ops, particularly

File: Docs.md [hunk 3]
```diff
@@ -337,7 +337,22 @@ https://github.com/denoland/deno/blob/master/src/msg.fbs
 See
 [CONTRIBUTING.md](https://github.com/denoland/deno/blob/master/.github/CONTRIBUTING.md).
 
-## Change Log
+## Changelog
+
+### 2018.11.27 / v0.2.0 / Mildly usable
+
+The resources table was added to abstract various types of I/O streams and other
```
Comment (piscisaureus) [inline Docs.md]: This is not a user visible change - maybe place it at the end?

File: Docs.md (hunk 1 above)
Comment (ry) [reply Docs.md]: Fixed.

File: Docs.md (hunk 3 above)
Comment (ry) [reply Docs.md]: fixed

File: Docs.md (hunk 2 above)
Comment (ry) [reply Docs.md]: fixed

**Pull request merged**